docs/how-to/deploy_cups_sdcore.md

```diff
@@ -0,0 +1,217 @@
+# Deploy SD-Core with CUPS
+
+This guide covers how to install a multi-node sdcore with separation of user and control plane traffic.
+
+### Requirements
```

Comment:
As warned when running `make run`, there is a non-consecutive header level increase; H1 to H3. This applies at a couple of places.

Reply:
Done